### PR TITLE
Assign users to course

### PR DESCRIPTION
**Pull request by thiennv57**

### CODE REVIEW

File: app/views/admin/assign_users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+              checked: @course.users.include?(user)}, user.id, nil %>
+            <li><span><%= user.name %></span></li>
+          <% end %>
+        <% end %>
+      </ol>
+      <%= f.submit t("buttons.admin.course.update"), class: "btn btn-primary" %>
+    <% end %>
+  </div>
+</div>
+
```
Comment (ThuBM) [inline app/views/admin/assign_users/edit.html.erb]: remove line

**Comment by ThuBM** [comment]
OK

**Comment by VuongLong** [comment]
ok

**Comment by binhthanhnguyen** [comment]
okay

**Pull request merged**